x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/conversations/field_maps_configuration.ts

```diff
@@ -147,6 +147,11 @@ export const conversationsFieldMap: FieldMap = {
     array: false,
     required: false,
   },
+  'api_config.action_type_id': {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
```

Comment:
IIRC, there's no migration or specific schema versioning changes we need to do when adding new fields, right? From what I'm seeing the templates get updated on restart and things are good to go from there.